Vagrantfile

```diff
@@ -6,6 +6,10 @@ VIRTUAL_MACHINES = {
     :hostname => 'ansible-st2-ubuntu16',
     :box => 'ubuntu/xenial64',
   },
+  :ubuntu18 => {
```

Comment:
By adding U18 platform, it would be nice to make it default platform in Vagrant like `:ubuntu16` previous was.

So we'll need additional Vagrantfile and README.md updates.